.editorconfig

```diff
@@ -1,13 +1,17 @@
 # Stop .editorconfig files search on current file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
+indent_style = space
+indent_size = 2
 end_of_line = lf
-insert_final_newline = true
+charset = utf-8
 trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 2000
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
 
-# Indentation override
-[*]
-indent_style = space
-indent_size = 2
+[src/**]
+max_line_length = 120
```

Comment:
See above. Also prettier, eslint and vscode are all configured to use `100` characters max.

Reply:
Should I remove this line?